### PR TITLE
Presence | FiveM

### PR DESCRIPTION
**Pull request by marinofranz**
image
image

### CODE REVIEW

File: websites/F/FiveM/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+}
+
```
Comment (Bas950) [inline websites/F/FiveM/tsconfig.json:7]: remove the extra enters

File: websites/F/FiveM/presence.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+  clientId: "730897382937591848"
+});
+
+let browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let debugMode = true;
```
Comment (Bas950) [inline websites/F/FiveM/presence.ts:7]: merge lets

File: websites/F/FiveM/presence.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+presence.on("UpdateData", () => {
+  const presenceData: PresenceData = {
+      largeImageKey: "fivem",
+      startTimestamp: browsingTimestamp        
+  };
+
+  const pathname = document.location.pathname
+  const hostname = document.location.hostname;
```
Comment (Bas950) [inline websites/F/FiveM/presence.ts:20]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by marinofranz** [comment]
will fix later today

**Comment by marinofranz** [comment]
uh, codacy is detecting something that's not in my code

File: websites/F/FiveM/presence.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+const presence = new Presence({
+  clientId: "730897382937591848"
+});
+
+const browsingTimestamp = Math.floor(Date.now() / 1000),
```
Comment (Bas950) [inline websites/F/FiveM/presence.ts:5]: merge these consts and then codacy should also update.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/F/FiveM/presence.ts (hunk 4 above)
Comment (marinofranz) [reply websites/F/FiveM/presence.ts]: resolved

**Comment by marinofranz** [comment]
codacy still having some issues

**Review by Alanexei — CHANGES_REQUESTED** [review]
@marinofranz Can you fix the changes that codacy suggests? This pull request has 2 more days left before it gets closed. If you need assistance, ping me in the Discord server.

**Comment by Bas950** [comment]
just update your fork, will probs fix the issues.

**Comment by marinofranz** [comment]
fork updated

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Comment by a6doff** [comment]
Update: this broke the build

**Comment by a6doff** [comment]
```
websites/F/FiveM/presence.ts (1,7): Cannot redeclare block-scoped variable 'presence'.
websites/F/FiveM/presence.ts (6,1): Cannot redeclare block-scoped variable 'presence'.
websites/F/FiveM/presence.ts (6,1): Variable 'presence' implicitly has an 'any' type.
websites/F/FiveM/presence.ts (6,9): ',' expected.
websites/F/FiveM/presence.ts (6,10): Variable 'on' implicitly has an 'any' type.
websites/F/FiveM/presence.ts (6,12): ',' expected.
websites/F/FiveM/presence.ts (6,13): Left side of comma operator is unused and has no side effects.
SyntaxError: /home/runner/work/Presences/Presences/websites/F/FiveM/dist/presence.js: Identifier 'presence' has already been declared (3:55)
```

**Pull request merged**